bindings/python/setup.py

```diff
@@ -38,15 +65,21 @@ def bjam_build():
         # on windows, just link all the dependencies together to keep it simple
         toolset += ' libtorrent-link=static boost-link=static'
 
+    # statically link libtorrent when building wheel
+    if 'bdist_wheel' in sys.argv and platform.system() != 'Windows':
+        toolset += ' libtorrent-link=static'
+
     parallel_builds = ' -j%d' % multiprocessing.cpu_count()
     if sys.maxsize > 2**32:
```

Comment:
is this reasonable? if we "cross-compile" a 32 bit build on a 64 bit system, this wouldn't work, right?

Reply:
Shouldn't Boost then recognize that we are building 32 bit build and also build and link libtorrent in 32 bits? It works with tests on CI which is 64 bit with 32 bit Python but I didn't test this on actual 32 bit system.

Reply:
Generally, the standard with building extensions with setuptools is that the extension should be compatible with the interpreter that invoked `setup.py`. So you'd "cross-compile" by installing and invoking a 32-bit python.